index.html

```diff
@@ -304,23 +305,9 @@
         </table>
       </div>   
           
-      <div id="id_sitefooter">
-        <table id="id_sitefooter_table0">
-          <script type="text/javascript" src="./scripts/static_text/sitefooter/content_sitefooter_table0.js">
+      <div id="id_sitefooter">	  
+        <script type="text/javascript" src="./scripts/static_text/sitefooter/content_sitefooter.js">
           </script>
-          <!--v   Open Row0   v-->
-          <tr id="id_sitefooter_table0_row0">
-            <script type="text/javascript" src="./scripts/static_text/sitefooter/content_sitefooter_table0_row0.js">
-            </script>
-            <!--v   Open Row0 Column0   v-->
-            <td id="id_sitefooter_table0_row0col0">
-              <script type="text/javascript" src="./scripts/static_text/sitefooter/content_sitefooter_table0_row0col0.js">
-              </script>
-            </td>
-            <!--^   Close Row0 Column0   ^-->
-          </tr>
-          <!--^   Close Row0   ^-->
-        </table>
       </div>
 
 	  </div>	
```

Comment:
(Review comment 3) Any thoughts on importing all JS files in one place down below? Right now the imports are contained within their HTML so that developer can more easily verify that there is a one-to-one mapping between that specific location in the HTML (e.g. by unique ID) and the static content file (currently a JS file) for that location. Would like to discuss pros/cons of each approach